app/src/main/res/values/colors.xml

```diff
@@ -337,4 +337,7 @@
 
     <!-- App Spinners colors -->
     <color name="spinner_selected_item">#1415141A</color>
+
+    <!-- Material Design colors -->
+    <color name="material_scrim_color">#52000000</color>
```

Comment:
We should have a design system scrim color

Reply:
The app already uses a custom scrim color for the share screen but that has a blue tint.
The value I added here is a a black color with 32% opacity which seems to mimic exactly what the scrim would be when using a dialog and is a value recommended in the material design specs - https://m2.material.io/develop/android/theming/motion